packages/site/landing/index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv='X-UA-Compatible' content='IE=edge'>
```

Comment:
Wanna update the example apps as well?

angular:
https://github.com/ni/nimble/blob/b39cac6d5651afee85bd9f5e18c91d2e7ad17503/angular-workspace/projects/example-client-app/src/app/app.module.ts#L69

blazor: 
https://github.com/ni/nimble/blob/b39cac6d5651afee85bd9f5e18c91d2e7ad17503/packages/nimble-blazor/Examples/Demo.Client/wwwroot/index.html#L7
https://github.com/ni/nimble/blob/b39cac6d5651afee85bd9f5e18c91d2e7ad17503/packages/nimble-blazor/Examples/Demo.Shared/Shared/MainLayout.razor#L6

Maybe:
Angular Kitchen Sink Demo - Nimble Design System - NI
Blazor Kitchen Sink Demo - Nimble Design System - NI

Reply:
I think "kitchen sink" demo is a well-understood term, but don't feel strongly. If the term "kitchen sink" isn't desired can you leave a comment to discuss update it in the examples draft hld: https://github.com/ni/nimble/pull/827/files#diff-0967e47790c3f683cc13aff0bdc76ecdcf42ebe20f0d9aadcb89b4fe7bf2ed5aR70

Reply:
I left a comment over there 👍